components/places/Cargo.toml

```diff
@@ -10,7 +10,7 @@ default = []
 
 [dependencies]
 sync15 = { path = "../sync15" }
-serde = "1.0.75"
+serde = { version = "1.0.75", features = ["derive"] }
```

Comment:
We shouldn't add derive and still depend on serde_derive.

Also lets not add this until we can add it globally, since it ends up slowing down compilation because everything depends on serde, and so turning on/off features will cause it to need to be recompiled, following with all things that depend on it.